src/util/persisted-data.test.ts

```diff
@@ -1,94 +1,106 @@
 /* eslint-env jest */
-import fs from 'fs-extra-promise';
-import jestFs from 'jest-plugin-fs';
-import yaml from 'js-yaml';
+import fs from 'fs-extra';
+
 import { isEqual } from 'lodash';
+import path from 'path';
 
 import { IRepo } from '../adapters/base';
 import { IMigrationContext } from '../migration-context';
 import { loadRepoList, updateRepoList } from './persisted-data';
 
-jest.mock('fs', () => require('jest-plugin-fs/mock')); // eslint-disable-line global-require
-
-const reposFixture = [{
-  owner: 'NerdWallet',
-  name: 'test',
-}];
 
-const makeContext = () => ({
+const makeContext = (workingDirectory: string) => ({
   migration: {
-    workingDirectory: '/migration',
+    workingDirectory,
   },
   adapter: {
     reposEqual: (r1: IRepo, r2: IRepo) => isEqual(r1, r2),
   },
 } as IMigrationContext);
 
 describe('persisted-data', () => {
-  beforeEach(() => jestFs.mock({
-    '/migration/repos.json': JSON.stringify(reposFixture),
-  }));
-  afterEach(() => jestFs.restore());
 
   it('loads repo list from a file', async () => {
-    const repos = await loadRepoList(makeContext());
-    expect(repos).toEqual([{ owner: 'NerdWallet', name: 'test' }]);
+    const workingDirectory = path.join(__dirname, '../../fixtures/artifacts/.shepherd/load-repo-from-file/');
+    const repos = await loadRepoList(makeContext(workingDirectory));
+    // fs.unlinke(path.join(workingDirectory,'/repos.json'));
```

Comment:
Can you remove this line if it's unnecessary?

Reply:
Will do.